english/critter/shoo/shoo.txt

```diff
@@ -0,0 +1,2 @@
+You wave your hands at the citter and it freezes, then backs away, blinking at you confusedly. You tell yourself 
+that at least you didn't get bit by a strange critter today, and besides, you'll probably need that extra sandwich later.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.